app/src/main/java/org/oppia/app/parser/StringToFractionParser.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.app.parser
+
+import org.oppia.app.model.Fraction
+
+/** This class contains method that helps to parse string to fraction. */
+class StringToFractionParser {
```

Comment:
Added StringToFractionParser class in app module as it requires access to Fraction model, which wont be there in utility module